manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -316,6 +316,7 @@ properties:
   diego:
     ssl:
       skip_cert_verify: true
+      starting_container_weight: 0.25
```

Comment:
It would be good to add a reference to the commit message that explains why this change is necessary.